### ISSUE TITLE
undefined is not an object (evaluating '_react3.default.PropTypes.string')

### ISSUE THREAD

**Issue by GabrielBB** [MichaelCereda/react-native-form-generator#123]
I just copy the example code on the README.md and it throws this error. I think it's because something has changed in newer versions of React Native.

**Comment by hans0low** [comment]
Hey @GabrielBB I made a pull request today that goes through fixing the error you are seeing. React switched over to a different package called prop-types and that breaks it if you have a newer version.

**Comment by ccostel** [comment]
Did this get solved?

**Comment by hans0low** [comment]
@ccostel I made a PR for it but i don't think it was merged yet.

**Comment by ccostel** [comment]
I actually implemented your changes and they work. Hopefully @MichaelCereda  will soon merge it. 
Thanks for the PR :) 👍

**Comment by saxahan** [comment]
Still undefined is not an object (evaluating '_react3.default.PropTypes.string'). Did you update repo?

**Comment by lakshyaGoel** [comment]
Yeah, I also had to apply the changes of the PR manually, But it works. Thanks @hans0low

**Comment by schoetlr** [comment]
I just installed this library today and am getting this error.

**Comment by aanchan** [comment]
@ccostel @hans0low is there a way of applying the PR automatically while waiting for the merge to happen? i tried suggestions [here](https://stackoverflow.com/questions/33181297/npm-install-from-github-pull-request) but kept running into errors.

**Comment by hans0low** [comment]
npm install MichaelCereda/react-native-form-generator#pull/124/head

I believe this should work.

**Comment by ccostel** [comment]
It's been a while and @MichaelCereda still did not merge. I tweeted him in the hope that it will come to his attention.

**Comment by ventocilla** [comment]
Hi, 

I'm getting error. I tried npm install MichaelCereda/react-native-form-generator#pull/124/head
but I'll receiveing the following message:

npm ERR! code 1
npm ERR! Command failed: /usr/bin/git checkout pull/124/head
npm ERR! error: pathspec 'pull/124/head' did not match any file(s) known to git.

I'm trting to run this using Expo. Can this be the problem ?

I ask for help on this issue.

**Comment by Mr-Recalculate** [comment]
As of now, the PR has been merged but after I got the same "undefined is not an object (evaluating '_react3.default.PropTypes.string')"

**Comment by Gabriel-J-Young** [comment]
@ventocilla You might want to try manually changing the files.

As of now, the @hans0low PR has been merged. But I installed the package today, then got the "undefined is not an object (evaluating '_react3.default.PropTypes.string')" error. I took a look at the .js files that the @hans0low PR  (#124)   changed but my local files (that I installed today), did not have the @hans0low PR changes applied. To fix the error, I had to manually edit my local .js files to match the @hans0low  PR. #

**Comment by hopewise** [comment]
I just installed the package today, I got this error which appears only if I only do:

    import { Form } from 'react-native-form-generator'

![Screen Shot 2019-04-22 at 1 03 47 PM](https://user-images.githubusercontent.com/1061157/56495700-3929f580-64ff-11e9-8307-dafe3d567526.png)

**Comment by saravanabalagi** [comment]
@hopewise did you manage to fix it?

**Comment by hopewise** [comment]
no, I abandoned using this package..
